### PR TITLE
Sven's feedback

### PR DESCRIPTION
**Pull request by svenvanderburg**
I have read through the lesson material (up till chapter 3) and added some feedback.

In general I am very positive! I like the pace and style, and I think it covers the right subjects.

### CODE REVIEW

File: _episodes/01-introduction.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 title: "Introduction"
 teaching: 20
-exercises: 15
+exercises: 5
```
Comment (svenvanderburg) [inline _episodes/01-introduction.md:4]: I would say that the recipe exercise wouldn't take more than a few minutes.

File: _episodes/03-computing-pi.md [hunk 2]
```diff
@@ -133,7 +146,7 @@ Numba makes it easier to create accellerated functions. You can use it with the
 import numba
 
 @numba.jit
-def sum_range(a: int):
+def numbified_sum_range(a: int):
```
Comment (jhidding) [inline _episodes/03-computing-pi.md:149]: maybe `numba_sum_range` is a better name

File: _episodes/03-computing-pi.md [hunk 3]
```diff
@@ -87,6 +97,9 @@ We can demonstrate that this is much faster than the 'naive' implementation. Thi
 > - monolithic approach, less composable?
 {: .discussion}
 
+FIXME: Before this, the student has not learned about dask at all. It could
+be that students will struggle with this excercise, while the
+solution is actually very simple. It might be better to just show it?
```
Comment (jhidding) [inline _episodes/03-computing-pi.md:102]: We actually show the first `dask` examples in the benchmarking chapter. It should be explained there that `dask.array` mimics the `numpy` API. We could repeat that information here as a hint to the exercise.

File: _episodes/03-computing-pi.md (hunk 2 above)
Comment (svenvanderburg) [reply _episodes/03-computing-pi.md]: Agree.

File: _episodes/03-computing-pi.md (hunk 3 above)
Comment (svenvanderburg) [reply _episodes/03-computing-pi.md]: I've added this to the FIXME.

**Pull request merged**